packages/amplify-graphql-auth-transformer/src/utils/index.ts

```diff
@@ -51,7 +51,7 @@ export const getAuthDirectiveRules = (authDir: DirectiveWrapper, featureFlags: F
     }
   };
 
-  const { rules } = authDir.getArguments<{ rules: Array<AuthRule> }>({ rules: [] }, featureFlags);
+  const { rules } = authDir.getArguments<{ rules: Array<AuthRule> }>({ rules: [] }, null, options);
```

Comment:
given the recommendation above, we could just remove the `null` argument from here and include `isField` in options